recipes/conversation_view/attachment_carousel/attachment_carousel.vue

```diff
@@ -0,0 +1,263 @@
+<template>
+  <div
+    class="dt-attachment-carousel"
+    role="presentation"
+    @mouseenter="showCarouselArrows = true"
+    @mouseleave="showCarouselArrows = false"
+    @focusin="showCarouselArrows = true"
+    @focusout="showCarouselArrows = false"
+  >
+    <div
+      v-if="mediaList.length > 0"
+      ref="carousel"
+      class="dt-attachment-carousel--media-list"
+      :aria-label="attachmentAriaLabel"
+      @scroll="handleScroll"
+    >
+      <!-- Image list -->
+      <div
+        v-for="(mediaItem, index) in mediaList"
+        :key="`media-${index}`"
+        role="presentation"
+        class="d-ps-relative"
+        @focusin="closeButton(true, index)"
+        @focusout="closeButton(false, index)"
+        @mouseenter="closeButton(true, index)"
+        @mouseleave="closeButton(false, index)"
+      >
+        <dt-image-viewer
+          image-button-class="d-h64 d-w64 d-bar4 d-ba d-baw6 d-bc-subtle"
```

Comment:
make a custom class for this using tokens, we try not to use utility classes in dialtone vue anymore.

Reply:
changed it in both PRs.